brave-tests/src/test/java/brave/internal/recorder/PendingSpansClassLoaderTest.java

```diff
@@ -43,16 +43,16 @@
     }
   }
 
-  @Test public void unloadable_afterErrorReporting() {
-    assertRunIsUnloadable(ErrorReporting.class, getClass().getClassLoader());
+  @Test public void unloadable_afterOrphan() {
+    assertRunIsUnloadable(OrphanedContext.class, getClass().getClassLoader());
   }
 
-  static class ErrorReporting implements Runnable {
+  static class OrphanedContext implements Runnable {
     @Override public void run() {
       PendingSpans pendingSpans =
         new PendingSpans(Platform.get().clock(), new FinishedSpanHandler() {
           @Override public boolean handle(TraceContext context, MutableSpan span) {
-            throw new RuntimeException();
```

Comment:
pendingspans no longer takes responsibility of error-guarding. that's the duty of NoopAware... which is used both on finish and abandon paths now.